runtime/laos/src/weights/pallet_laos_evolution.rs

```diff
@@ -2,22 +2,22 @@
 //! Autogenerated weights for `pallet_laos_evolution`
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
-//! DATE: 2024-06-26, STEPS: `50`, REPEAT: `20`, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! DATE: 2024-06-27, STEPS: `2`, REPEAT: `2`, LOW RANGE: `[]`, HIGH RANGE: `[]`
 //! WORST CASE MAP SIZE: `1000000`
-//! HOSTNAME: `trujideb`, CPU: `12th Gen Intel(R) Core(TM) i5-12500H`
+//! HOSTNAME: `titan`, CPU: `12th Gen Intel(R) Core(TM) i7-1260P`
 //! WASM-EXECUTION: `Compiled`, CHAIN: `Some("dev")`, DB CACHE: 1024
 
 // Executed Command:
 // ./target/release/laos
 // benchmark
 // pallet
 // --chain=dev
-// --steps=50
-// --repeat=20
+// --steps=2
```

Comment:
To increase the iterations as they were before would help to have a more robust benchmarks

Reply:
sure, I will when calculate the benchmarks for the release